src/node/CarplayNode.ts

```diff
@@ -15,16 +13,28 @@ import {
   DongleDriver,
   DongleConfig,
   DEFAULT_CONFIG,
+  Key,
+  CarPlay,
+  AudioCommand,
 } from '../modules'
 
-export default class CarplayWS extends EventEmitter {
+const USB_WAIT_PERIOD_MS = 500
+const USB_WAIT_RESTART_MS = 3000
+
+export type CarplayMessage =
```

Comment:
This type could now be moved to a shared folder so both node and web can reference it, but I kept it separate, just in case